Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionPublishToQCOperations.cs

```diff
@@ -163,7 +177,8 @@ private BusinessFlow ConvertRunSetToBF(IReportInfo reportInfo)
                         ActivitiesGroup virtualAG = new ActivitiesGroup();
                         virtualAG.Name = runSetBF.Name;
                         virtualAG.Description = runSetBF.Description;
-                        
+                        virtualAG.ExternalID = !string.IsNullOrEmpty(runSetBF.ExternalID) ? runSetBF.ExternalID : string.Empty;
+                        virtualAG.BusinessFlowGuid = runSetBF.Guid;
```

Comment:
parent Guid can be used here. add the comment what is being stored in it

Reply:
done